docs/reference.rst

```diff
@@ -544,7 +548,7 @@ LUA_prefix
 
 LUA_result
     The name of the Lua wrapper's result variable.
-    It defaults to *rv*  (return value).
+    It defaults to *SHT_<rv*  (return value).
```

Comment:
I've prefixed local variables with SH.  SHT is a Shroud Temporary.  The < is junk.  The case I'm trying to address is if a function has an argument named 'rv'.  This allows the user to change the name to avoid the conflict.